base/rational.jl

```diff
@@ -11,12 +11,19 @@ struct Rational{T<:Integer} <: Real
     den::T
 
     function Rational{T}(num::Integer, den::Integer) where T<:Integer
-        num == den == zero(T) && __throw_rational_argerror(T)
-        num2, den2 = signbit(den) ? divgcd(-num, -den) : divgcd(num, den)
-        new(num2, den2)
+        num == den == zero(T) && __throw_rational_argerror_zero(T)
+        # issue #32569
```

Comment:
Usually we don't insert references to issues in code (as opposed to tests), and here I think we can understand the reason of this test by following the thrown error, faster than by looking up github. No really a problem though, and probably not necessary to update your PR.

Reply:
That sounds reasonable. Went ahead and deleted it now.